scripts/rest_examples/reservation-create-aws.http

```diff
@@ -1,10 +1,15 @@
 // @no-log
-POST http://{{hostname}}:{{port}}/{{prefix}}/reservations/aws HTTP/1.1
+POST http://localhost:8005/api/provisioning/v1/reservations/aws HTTP/1.1
 Content-Type: application/json
 X-Rh-Identity: {{identity}}
 
 {
+
+  "source_id": 9, 
+  "image_id": "ami-0cff7528ff583bf9a",
+  "amount": 1,
+  "instance_type": "t1.micro",
   "pubkey": {
-    "existing_id": 1
+    "existing_id": 2
```

Comment:
Can you keep this on 1? Because if you seed the database it is the only key that is present (my key). Feel free to add more keys into the seed tho. Then make this a `{variable}` so it can be configured locally.